lib/package/plugins/policyMetaData.json

```diff
@@ -38,7 +38,7 @@
   "OAuthV1" :{"indications":["oauthv1","oauth","oa","accesstoken","verify"], "profiles":["apigee"] },
   "OAuthV2" :{"indications":["oauthv2","oauth","oa","accesstoken","verify"]},
   "OASValidation" :{"indications":["oas","oasValidation","^oas$"]},
-  "ParseDialogRequest":{"indications":["parsedialogrequest","dialogflow","pdr","^parsedialogrequest$"], "profiles":["apigeex"] },
+  "ParseDialogflowRequest":{"indications":["parsedialogrequest","dialogflow","pdr","^parsedialogrequest$"], "profiles":["apigeex"] },
```

Comment:
`{"indications":["parsedialogrequest","dialogflow","pdr","^parsedialogrequest$"], "profiles":["apigeex"] }`

should we rather do
`{"indications":["parsedialogflowrequest","dialogflow","pdr","^parsedialogflowrequest$"], "profiles":["apigeex"] }`

Reply:
done